<launch>
  <!-- Launch file for velodyne16 VLP-16 LiDAR -->

    <arg name="rviz" default="true" />

    <rosparam command="load" file="$(find sr_livo)/config/r3live_compressed.yaml" />

    <param name="debug_output" type="bool" value="0"/>
    <param name="output_path" type="string" value="$(find sr_livo)/output"/>
    <node pkg="sr_livo" type="livo_node" name="livo_node" output="screen" /> 

    <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find sr_livo)/rviz_cfg/visualization.rviz" />
    </group>

</launch>
